### PR TITLE
WD-14793 - add new /azure/confidential-computing page

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Add new /azure/confidential-computing page according to the copy doc and figma

## QA
- [demo link](https://ubuntu-com-14365.demos.haus/azure/confidential-computing)
- [copy doc](https://docs.google.com/document/d/1sVaQQlm6LHyRf-Q3VSVTmqGRi0SV7gwBVczCTPWq3CE/edit#heading=h.3hmfjtayfrqm)
- [figma](https://www.figma.com/design/MinLlXouBnEDqaoAL3bTWC/24.10-ubuntu.com%2Fazure?node-id=543-1498&node-type=frame&t=GMkgEW8xbup8LJql-0)
- Check the site on the demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to copy doc and figma

## Issue / Card
[WD-14793](https://warthogs.atlassian.net/browse/WD-14793)

[WD-14793]: https://warthogs.atlassian.net/browse/WD-14793?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mattea-turic** [comment]
A couple of things looked a little off in the figma file, so I made some adjustments (discussed on mm).
Assets server can be found [here](https://assets.ubuntu.com/manager?tag=confidential-computing-page)

**Comment by Sophie-32** [comment]
@lizzochek - bullets are missing from the "Get Ubuntu confidential VMs on the Microsoft Azure marketplace" section on the demo

**Comment by mattea-turic** [comment]
could you add shallow padding between the image and text here:
Screenshot 2024-09-30 at 13 30 10

the image there with the azure logo was also a off by a few pixels, so I realigned it now so it should be centred! Added [here](https://assets.ubuntu.com/v1/bee89574-azure-logo-updated.png)

add break here after "GPUs" pls so second line reads "on Microsoft Azure"
Screenshot 2024-09-30 at 13 37 24

break after "about"
Screenshot 2024-09-30 at 13 38 36

File: templates/azure/confidential-computing.html [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+{% block content %}
+  <section class="p-section--hero p-suru">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>Confidential computing with Ubuntu on Microsoft Azure</h1>
+      </div>
+      <div class="col">
+        <p>Secure your data, code and AI models at run- time with Ubuntu confidential VMs on Microsoft Azure.</p>
+        <hr class="p-rule--muted" />
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=Canonical%20Ubuntu%20confidential%20VM&page=1">Try Ubuntu confidential VMs today&nbsp;&rsaquo;</a>
```
Comment (britneywwc) [inline templates/azure/confidential-computing.html:26]: Please wrap the anchor tag in a `p`, this was a comment that I received from MPT from a previous PR

File: templates/azure/confidential-computing.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+          </li>
+          <li class="p-list__item has-bullet">
+            <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-confidential-vm-jammy">Get AMD SEV SNP with Ubuntu 22.04 LTS on Azure</a>
+          </li>
+          <li class="p-list__item has-bullet">
+            <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-confidential-vm-focal?tab=Overview">Get AMD SEV SNP with Ubuntu  20.04 LTS on Azure</a>
+          </li>
+        </ul>
+        <hr class="p-rule--muted" />
+        <a href="/confidential-computing#get-in-touch">Don’t see what you’re looking for? Contact us to learn more&nbsp;&rsaquo;</a>
```
Comment (britneywwc) [inline templates/azure/confidential-computing.html:82]: Similarly here, can you please wrap it in a `p`?

File: templates/azure/confidential-computing.html [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Use Ubuntu Pro to further harden your confidential VMs</h2>
+      </div>
+      <div class="col">
+        <p>
+          While confidential VMs can protect your workload from external threats, vulnerabilities from within their boundaries remain a concern. With Ubuntu Pro, you can keep your guest CVM software stack continuously patched and up-to-date for up to 12 years, covering 30,000 open source packages with a single subscription.
+        </p>
+        <hr class="p-rule--muted" />
+        <a href="/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a>
```
Comment (britneywwc) [inline templates/azure/confidential-computing.html:98]: This should be wrapped in `p`

File: templates/azure/confidential-computing.html [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        <a href="https://ubuntu.com/confidential-computing">Learn about other confidential computing offerings from Canonical&nbsp;&rsaquo;</a>
+      </h2>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
```
Comment (britneywwc) [inline templates/azure/confidential-computing.html:115]:
```suggestion
    <div class="u-fixed-width">
```
Design doesn't have shallow padding for the header row

File: templates/azure/confidential-computing.html [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+      <div class="col">
+
+        <p class="u-no-padding--top">
+          Businesses using machine learning on Azure are prioritizing data security and model protection. Strict regulations often prevent sharing sensitive information, holding back AI’s full capabilities in key sectors.
+        </p>
+        <p>
+          To address this challenge on Azure, we introduce a confidential AI preview featuring Ubuntu VMs with AMD 4th Gen EPYC processors and NVIDIA H100 GPUs, delivering stronger security and performance for advanced AI workloads.
+        </p>
+        <hr class="p-rule--muted" />
+        <a href="https://aka.ms/accgpusignup">Sign up for the Azure preview of confidential AI with Ubuntu&nbsp;&rsaquo;</a>
```
Comment (britneywwc) [inline templates/azure/confidential-computing.html:140]: Wrap in `p` as well

**Review by britneywwc — CHANGES_REQUESTED** [review]
Left some comments, can you also add the new page to sitemaps? Thanks

**Comment by britneywwc** [comment]
Thanks for making the changes, can you please add the new page to the sitemaps? I'll leave a +1 once that's done :)

**Review by britneywwc — APPROVED** [review]

**Pull request merged**